iroh-net/Cargo.toml

```diff
@@ -16,6 +16,7 @@ rust-version = "1.76"
 workspace = true
 
 [dependencies]
+async-trait = "0.1.77"
```

Comment:
I think we don't need this anymore, so let's not forget removing it :)

Reply:
bless